lib/shared/bucketing-assembly-script/__tests__/bucketing/segmentation.test.ts

```diff
@@ -394,6 +395,184 @@ describe('SegmentationManager Unit Test', () => {
             })
         })
 
+        describe('evaluateOperator should handle audienceMatch filter', () => {
```

Comment:
it would be good to also have at least one test that tests this filter from the entry point of segmentation where the config data is passed. Currently we aren't covering the earlier code paths that pass the audience through to `evaluateOperator`

Reply:
yeah good callout, ill add a test in the bucketing.test file that tests the output of a bucketed config